web/client/reducers/geostory.js

```diff
@@ -189,6 +205,24 @@ export default (state = INITIAL_STATE, action) => {
         case SET_CURRENT_STORY: {
             return set('currentStory', action.story, state);
         }
+        case SET_CONTROL: {
+            const {control, value} = action;
+            return set(`controls.${control}`, value, state);
+        }
+        /**
+         * **NOTE** this is the resource that contains the whole story (e.g. GeoStore).
+         * It contains permissions and so on. Don't confuse with story resources (media).
+         */
+        case SET_RESOURCE: {
+            const { resource } = action;
+            return set(`resource`, resource, state);
```

Comment:
what about using storyResource,  backendResource, externalResource?

Reply:
I wanted to follow the same name convention of dashboard. I documented this name conflict but I don't think is a problem by itself. The two resources concepts are very different, and the signature is different too.

The only problem may concern the action names, but I think is more correct to rename the story resources (media) actions instead.

I should postpone this when working more on story resources. What do you think?